pkg/controller/awselb/aws_elb_controller.go

```diff
@@ -285,6 +325,36 @@ func (c *Controller) syncMachine(key string) error {
 		return err
 	}
 
+	return c.updateStatus(machine, mLog)
+}
+
+// updateStatus sets the LastELBSync time in status to now, and updates the machine.
+func (c *Controller) updateStatus(machine *capiv1.Machine, mLog log.FieldLogger) error {
+
+	mLog.Debug("updating machine status")
+
+	awsStatus, err := controller.AWSMachineProviderStatusFromClusterAPIMachine(machine)
+	if err != nil {
+		return err
+	}
+	awsStatus.LastELBSync = &metav1.Time{Time: time.Now()}
```

Comment:
A nit, but I would rather see this written like this.

```
now := metav1.Now()
awsStatus.LastELBSync = &now
```

Then, you can also reuse `now` later when setting `LastUpdated`.